.github/workflows/self-update-on-new-release-version.yaml

```diff
@@ -0,0 +1,56 @@
+name: Self update on new release
+on:
+  push:
+    branches:
+      - release-*
+
+jobs:
+  update-e2e-ondemand-release:
```

Comment:
this is really similar to the Renovate workflow we have, so we should combine those and only have a single PR when a new release branch is created.

Reply:
it's a good idea, if you want I can do this way, just was not sure do we want same workflow or different.

Reply:
we should only have a single workflow that updates files for the release, so let's combine it

Reply:
do you have idea how to name it? 
`new-release`?

Reply:
fixed in 4d1a38e0